.github/workflows/iroha2-dev.yml

```
@@ -19,75 +23,52 @@ jobs:
       - uses: actions/checkout@v2
       - uses: Swatinem/rust-cache@v1
       - name: set up buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
         id: buildx
         with:
           install: true
-      - name: Login to DockerHub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Build and push docker image
-        uses: docker/build-push-action@v2
+          driver: docker
+
+      - name: Build iroha2:base image
+        uses: docker/build-push-action@v3
         with:
+          load: true
+          tags: iroha2:base
           context: .
-          push: true
-          tags: hyperledger/iroha2:dev
-          build-args: |
-            TARGET_DIR=release
-            PROFILE=--release
-      - name: Build and push docker image (iroha_client_cli)
-        uses: docker/build-push-action@v2
+          file: Dockerfile.ci        
+
+      - name: Build iroha2:build docker image
+        uses: docker/build-push-action@v3
         with:
+          load: true
+          tags: iroha2:build
           context: .
-          push: true
-          tags: hyperledger/iroha2:client-cli-dev
-          build-args: |
-            TARGET_DIR=release
-            PROFILE=--release
-            BIN=iroha_client_cli
-      - name: Build and push docker image (kagami)
-        uses: docker/build-push-action@v2
+          file: Dockerfile.build
+
+      - name: build iroha2:dev docker image
```

```suggestion
      - name: Build iroha2:dev docker image
```

The same for `iroha2.yml`